custom_model_runner/datarobot_drum/drum/perf_testing.py

```diff
@@ -266,8 +278,8 @@ def _stop_server(self):
         except (requests.exceptions.ReadTimeout, requests.exceptions.ConnectionError):
             pass
         finally:
-            time.sleep(0.5)
-            self._server_process.kill()
+            os.kill(self._server_process.pid, signal.SIGINT)
+            time.sleep(1)
```

Comment:
whats this do

Reply:
This is for production mode, where mlpiper catches SIGINT and gracefully stops uwsgi and nginx.
But drum also catches SIGINT and does `os._exit()`

Reply:
But why 1 second?

Reply:
What would you suggest?

Reply:
deleting the statement.

Reply:
**DONE** tested, looks ok